bitrise/setup.go

```diff
@@ -13,18 +13,20 @@ import (
 	"github.com/bitrise-io/go-utils/colorstring"
 )
 
+type SetupMode int
+
 const (
```

Comment:
NIT: I would put the 2 new setup modes into a separate const block (after the SetupMode type definition) because, at first sight, I got confused if envman or stepman versions are related to the step mode or not.

I was also surprised that iota works when there are const with different type in the same const block.